lib/utils/ui/user_avatar.dart

```diff
@@ -0,0 +1,41 @@
+import "package:flutter/material.dart";
+
+/// A widget that displays the user's avatar.
+/// It provides a [onTap] parameter to handle the user's tap,
+/// which adds an InkWell response.
+class UserAvatar extends StatelessWidget {
+  const UserAvatar({
+    super.key,
+    required this.displayName,
+    required this.radius,
+    this.onTap,
+  });
+
+  final String displayName;
+  final double radius;
+  final void Function()? onTap;
```

Comment:
You can use `VoidCallback?` here.

Reply:
You are right ! I wasn't aware of this typedef, thank you !

Added in: [06bbad6](https://github.com/ProximaEPFL/proxima/pull/158/commits/06bbad6a54607044acbe9f7d67c31a56409fa3d9)